mne_bids/tests/test_mne_bids.py

```diff
@@ -64,6 +64,16 @@ def test_fif():
     write_raw_bids(raw, bids_basename, output_path, events_data=events_fname,
                    event_id=event_id, overwrite=False)
 
+    # check if BIDS works without stim channel
+    raw.set_channel_types({raw.ch_names[i]: 'misc'
+                           for i in
+                           mne.pick_types(raw.info, stim=True, meg=False)})
+    output_path = _TempDir()
+    write_raw_bids(raw, bids_basename, output_path, overwrite=False)
+
+    cmd = ['bids-validator', output_path]
+    run_subprocess(cmd, shell=shell)
```

Comment:
Why do we need this test? The comment `check if BIDS works without stim channel` sounds like something to check in the specification, not as a test in software code?

Reply:
okay I need to change the comment then :) If you had no stim channel, then mne-bids was failing before ...